### ISSUE TITLE
ESLint 6: Can't load plugins when node_modules is not at the project root

### ISSUE THREAD

**Issue by hathawsh** [SublimeLinter/SublimeLinter-eslint#281]
Upon upgrading my create-react-app project to ESLint 6.2.1, I started getting this error from SublimeLinter-eslint:

```
Oops! Something went wrong! :(

ESLint: 6.2.1.

ESLint couldn't find the plugin "eslint-plugin-react".

(The package "eslint-plugin-react" was not found when loaded as a Node module from the directory "/home/shane/scm/opnreco".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install eslint-plugin-react@latest --save-dev

The plugin "eslint-plugin-react" was referenced from the config file in "frontend/.eslintrc.js".

If you still can't figure out the problem, please stop by https://gitter.im/eslint/eslint to chat with the team.
```

This error only happened within Sublime Text. When I used eslint at the command line, everything worked. I was rather confused until I noticed that eslint is looking for plugins in the wrong directory: it should be looking in ``/home/shane/scm/opnreco/frontend``, but it's looking in ``/home/shane/scm/opnreco``. My Sublime project folder is ``/home/shane/scm/opnreco``, but the Node project part of the project is at ``/home/shane/scm/opnreco/frontend`` (and node_modules is at ``/home/shane/scm/opnreco/frontend/node_modules``.)

ESLint 5 apparently knew how to find the local plugins regardless of the current working directory, while ESLint 6 just looks for plugins in the current working directory (which is ``/home/shane/scm/opnreco``) unless the ``--resolve-plugins-relative-to path`` parameter is passed to eslint.

My guess is SublimeLinter-eslint should set the working directory for the ``eslint`` command to the directory containing ``node_modules``. That way SublimeLinter-eslint doesn't need to know the version of eslint it's calling.

**Comment by hathawsh** [comment]
BTW, as a quick workaround, I made a symlink so eslint could find its plugins:

```
cd /home/shane/scm/opnreco
ln -s frontend/node_modules
```

This isn't very clean because the project root should not have any Node artifacts, but it's good enough for now.

**Comment by boutils** [comment]
I got the exact same issue (for a few days) and used the same workaround.

**Comment by kaste** [comment]
Thanks for reporting! This looks like it will be fixed in upstream https://github.com/SublimeLinter/SublimeLinter/issues/1626

Don't know why you need the `ln` hack though. You could set the `working_dir` or massage `args` setting to include `--resolve-plugins-relative-to path` as well.

**Comment by kaste** [comment]
Yes sure, http://www.sublimelinter.com/en/stable/settings.html#project-settings

**Comment by kaste** [comment]
Fixed upstream in v 4.13.0
